internal/admin/server.go

```diff
@@ -26,6 +26,7 @@ const (
 var Mux = http.NewServeMux()
 
 var baseplateGoCollectors = collectors.WithGoCollectorRuntimeMetrics(
+	collectors.MetricsGC,
```

Comment:
I didn't realize MetricsAll was an option; can we just use MetricsAll?

Reply:
All is, IMO, a bit too noisy. I would prefer to be selective in what we enable.